source/iostreams.tex

```diff
@@ -4600,19 +4598,12 @@
 To decide if the character \tcode{c} is a whitespace character,
 the constructor performs as if it executes the following code fragment:
 \begin{codeblock}
-const ctype<charT>& ctype = use_facet<ctype<charT>>(is.getloc());
-if (ctype.is(ctype.space, c) != 0)
+const ctype<charT>& ct = use_facet<ctype<charT>>(is.getloc());
+if (ct.is(ct.space, c))
   // \tcode{c} is a whitespace character.
 \end{codeblock}
```

Comment:
I think these are good improvements so far.

These two paragraphs are saying the same thing, I think it would be even clearer if we combined them:

```suggestion
The currently imbued locale in \tcode{is} is used
to determine whether or not the next input character is whitespace.
The character \tcode{c} is a whitespace character if
\tcode{ct.is(ct.space, c)} is \tcode{true},
where \tcode{ct} is \tcode{use_facet<ctype<charT>>(is.getloc())}.
```